packages/p2p-v2/src/components/Modals/NicknameModal/NicknameModal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ComponentType, useEffect } from 'react';
+import { debounce } from 'lodash';
+import { Controller, useForm, useWatch } from 'react-hook-form';
+import Modal from 'react-modal';
+import { p2p } from '@deriv/api';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useDevice } from '../../../hooks';
+import P2PUserIcon from '../../../public/ic-cashier-p2p-user.svg';
+import { Input } from '../../Input';
+import { customStyles } from '../helpers';
+import './NicknameModal.scss';
+
+const NicknameModal = () => {
+    const ReactModal = Modal as ComponentType<ReactModal['props']>;
```

Comment:
```suggestion
    const ReactModal = Modal as ComponentType<ReactModal['props']>;
```

One more question @ameerul-deriv, why is this line needed? 🤔

Reply:
I was getting an issue that said the 'ReactModal' cannot be used as a JSX component. So I added the above